.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ["2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+        gemfile: ["4.2", "4.2.api", "5.0", "5.1", "5.2"]
```

Comment:
Same here most of these Rails versions are totally EOL.

Reply:
Yes, again, I think we should only test 5.2, 6.0, and 6.1.